tests/codemods/test_lazy_logging.py

```diff
@@ -0,0 +1,683 @@
+import pytest
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+from core_codemods.lazy_logging import LazyLogging
+
+logging_funcs = {"debug", "info", "warning", "error", "critical"}
+each_func = pytest.mark.parametrize("func", logging_funcs)
+
+
+class TestLazyLoggingModulo(BaseSemgrepCodemodTest):
+    codemod = LazyLogging
+
+    def test_name(self):
+        assert self.codemod.name == "lazy-logging"
+
+    @each_func
+    @pytest.mark.parametrize(
+        "input_code,expected_output",
+        [
+            (
+                """\
```

Comment:
Just a very small thing but because of the way that these test cases are automatically dedented, I don't think the line continuation is necessary here. It ends up looking a little cleaner without.